docs/sources/explore/explore-inspector.md

```diff
@@ -0,0 +1,83 @@
++++
+title = "Inspector in Explore"
+weight = 400
++++
+
+# Inspector in Explore
+
+The inspector helps you understand and troubleshoot your queries. You can inspect the raw data, export that data to a comma-separated values (CSV) file, export log results in TXT format, and view query requests.
+
+## Inspector UI
+
+The inspector has following tabs:
+
+- **Stats tab -** Shows how long your query takes and how much it returns.
+- **Query tab -** Shows you the requests to the server sent when Grafana queries the data source.
+- **JSON tab -** Allows you to view and copy the data JSON and data frame structure JSON.
+- **Data tab -** Shows the raw data returned by the query.
+- **Error tab -** Shows the error. Only visible when query returns error.
+
+## Inspector tasks
+
+You can perform a variety of tasks in the Explore inspector.
+
+### Open the Inspector
+
+1. Run the query you would like to inspect.
+1. Click the **Inspector** button.
+
+The inspector pane opens on the bottom of the screen.
+
+### Inspect raw query results
+
+You can view raw query results, that is the data returned by the query in a table.
+
+1. In the **Inspector** tab, click the **Data** tab.
```

Comment:
Since this is just one step, it should be an indented paragraph. Not a numbered step.

Reply:
```suggestion
  In the **Inspector** tab, click the **Data** tab.
```

Reply:
By indented paragraph you mean this (See suggestion above)? @achatterjee-grafana

Reply:
Perfect. Thanks!